terracumber-cli

```diff
@@ -200,7 +209,7 @@ def run_terraform(args, tf_vars):
         terraform.taint(args.taint)
     if args.destroy:
         terraform.destroy()
-    result = terraform.apply()
+    result = terraform.apply(args.parallelism)
```

Comment:
It seems that I missed that when implementing the parallelism to terraform. Nice catch Maxime!